TotalCrossVM/docker/arm64/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM arm64v8/ubuntu:xenial
```

Comment:
The EOL for Xenial is 2021-04. The end of life will happen in about 8 months. After that, only security fixes.

Shouldn't be used a version with an extended life cycle? Like, Beaver or Fossa?

Reply:
Moved to a new issue